packages/@aws-cdk/aws-stepfunctions-tasks/README.md

```diff
@@ -9,4 +9,668 @@
 ---
 <!--END STABILITY BANNER-->
 
-See the README of the `@aws-cdk/aws-stepfunctions` library.
+[AWS Step Functions](https://docs.aws.amazon.com/step-functions/latest/dg/welcome.html) is a web service that enables you to coordinate the
+components of distributed applications and microservices using visual workflows.
+You build applications from individual components that each perform a discrete
+function, or task, allowing you to scale and change applications quickly.
+
+A [Task](https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-task-state.html) state represents a single unit of work performed by a state machine. All work in your state
+machine is performed by tasks.
+
+This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+
+## Table Of Contents
+
+- [Task](#task)
+  - [Parameters](#task-parameters-from-the-state-json)
+  - [Batch](#batch)
+    - [SubmitJob](#submitjob)
+  - [DynamoDB](#dynamodb)
+    - [GetItem](#getitem)
+    - [PutItem](#putitem)
+    - [DeleteItem](#deleteitem)
+    - [UpdateItem](#updateitem)
+  - [ECS](#ecs)
+    - [RunTask](#runtask)
+  - [EMR](#emr)
+    - [Create Cluster](#create-cluster)
+    - [Termination Protection](#termination-protection)
+    - [Terminate Cluster](#terminate-cluster)
+    - [Add Step](#add-step)
+    - [Cancel Step](#cancel-step)
+    - [Modify Instance Fleet](#modify-instance-fleet)
+    - [Modify Instance Group](#modify-instance-group)
+  - [Evaluate Expression](#evaluate-expression)
+  - [Glue](#glue)
+  - [Lambda](#lambda)
+  - [SageMaker](#sagemaker)
+    - [Create Training Job](#create-training-job)
+    - [Create Transform Job](#create-transform-job)
+  - [SNS](#sns)
+  - [Step Functions](#step-functions)
+  - [SQS](#sqs)
+
+### Task
+
+A `Task` represents some work that needs to be done. In the CDK, the exact work to be
+done is determine by a class that implements `IStepFunctionsTask`.
+
+AWS Step Functions [integrates](https://docs.aws.amazon.com/step-functions/latest/dg/concepts-service-integrations.html) with some AWS services so that you can call API
+actions, and coordinate executions directly from the Amazon States Language in
+Step Functions. You can directly call and pass parameters to the APIs of those
+services.
+
+#### Task parameters from the state JSON
+
+Many tasks take parameters. Parameter values can either be supplied
+directly in the workflow definition (by specifying their values), or at
+runtime by passing a value obtained from the static functions on `Data`,
+such as `Data.stringAt()`.
+
+If so, the value is taken from the indicated location in the state JSON,
+similar to (for example) `inputPath`.
```

Comment:
A short code snippet may be useful here before this statement.

Reply:
I dropped this statement since it's illustrated across examples. Might be worth supplementing this section with a few examples of `path` usage which is what this really tries to touch on.